Locales.lua

```diff
@@ -2039,6 +2039,16 @@ L["Emmah"] = true
 L["Box of Rattling Chains"] = true
 L["Sunreaver Micro-Sentry"] = true
 L["Haywire Sunreaver Construct"] = true
+L["Reins of the Winter Night Dreamsaber"] = true
+L["Reins of the Snowfluff Dreamtalon"] = true
+L["Reins of the Evening Sun Dreamsaber"] = true
+L["Reins of the Blossoming Dreamstag"] = true
+L["Reins of the Springtide Dreamtalon"] = true
+L["Reins of the Morning Flourish Dreamsaber"] = true
+L["Reins of the Rekindled Dreamstag"] = true
+L["This item drops from Emerald Bounties that started from Gigantic Seeds, it can also be purchased from a vendor."] =
```

Comment:
There's already a phrase for items that can be purchased from a vendor. If we use that, it doesn't have to be translated twice. Not sure about the Gigantic Seed notice - if this is something Rarity can detect we might as well remove the note. Otherwise, the explanation isn't clear as to what people should do, so players will have to look it up on wowhead anyway...?

Reply:
yeah pretty much, referring to the to it as epic seed is probably more obvious anyways because i see most referring to it as that